jenkinsfile.groovy

```diff
@@ -45,6 +45,11 @@ String manifestVersion = ""
 String assetsBasePath = ""
 String frontendArtifact = ""
 String backendArtifact = ""
+String documentsDir = "documents"
+String csvFileName = "RecallsFile.csv"
+String recallsTableName = "cvr-${params.environment}-recalls"
+String makesTableName = "cvr-${params.environment}-makes"
+String modelsTableName = "cvr-${params.environment}-models"
 net.sf.json.JSON manifestContent
```

Comment:
With introduction of /common It should be added to manifest and checked in order to determine whether we should rebuild all lambdas(similar to MOTR)

Reply:
This is resolved